src/motor_hardware.cc

```diff
@@ -62,6 +63,16 @@ int32_t  g_odomLeft  = 0;
 int32_t  g_odomRight = 0;
 int32_t  g_odomEvent = 0;
 
+// A 4wd robot must skid to turn. This factor approximates the actual rotation done vs what
+// the wheel encoders have indicated.  This only applies if in 4WD mode
+double   g_odom4wdRotationScale = 1.65;
+#define  WHEEL_VELOCITY_NEAR_ZERO   ((double)(0.05))
```

Comment:
To error on the side of caution I'll use 0.08
This parameter will become a ros param as soon as I can get an assist.   This will be a separate PR to make this a param along with another value so somebody can do that as a separate PR.  This will be later, not right now.